### PR TITLE
feat: appstudio: promote root image

### PR DESCRIPTION
**Pull request by psturc**
https://issues.redhat.com/browse/HACBS-788

### CODE REVIEW

File: ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+    ENV YQ_VERSION=v4.25.2
+
+    # Install yq and kubectl
+    RUN curl -Lso /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64 && chmod +x /usr/local/bin/yq
+    RUN curl -Lso /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x /usr/local/bin/kubectl
+  from: src
+  to: e2e-test-runner
+promotion:
+  name: redhat-appstudio
+  namespace: ci
```
Comment (psturc) [inline ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml:20]: should promote this image to openshift-ci registry after merge: `registry.ci.openshift.org/ci/redhat-appstudio:e2e-test-runner`

**Comment by psturc** [comment]
the rehearse job is failing on e2e test. however the issue is not related to this PR, because it doesn't change anything in the current e2e workflow.

@flacatus @jkopriva would you have a time to give this PR a look and add lgtm? Thanks

**Review by flacatus — APPROVED** [review]

**Comment by psturc** [comment]
I had to update this PR, because the prow config for build cluster for this job has changed. Could you please add another lgtm? Thanks
@flacatus @jkopriva

**Review by flacatus — APPROVED** [review]

**Pull request merged**